data/presets/public_transport/station_light_rail.json

```diff
@@ -8,6 +8,7 @@
     ],
     "geometry": [
         "point",
+        "vertex",
```

Comment:
The reason I proposed this change was less about the tagging style and more about the inconsistency:


https://github.com/openstreetmap/id-tagging-schema/assets/16009897/fd1cadd3-7c74-4e55-b680-73392194d845



If you drag a `Light Rail Station` onto a way, then the preset degenerates down to `Train Station`, which is pretty confusing. So ideally, all these presets would be allowed as verticies, or none of them. 

> I did not do any analysis how this is used because I lack the tooling to do this quickly.

[Here's an overpass query](https://overpass-turbo.eu/s/1NYg) to compare points vs vertices. There might be a more efficient query, but I'm not an overpass expert